org.eclipse.editorconfig.core/src/main/java/org/eclipse/editorconfig/internal/core/EditorConfigCorePlugin.java

```diff
@@ -0,0 +1,44 @@
+package org.eclipse.editorconfig.internal.core;

```

Comment:
I use unix line endings. I've updated the .editorconfig file on master to explicitly set line endings. If this is not a good solution then we can also use .gitattributes: https://help.github.com/articles/dealing-with-line-endings/